app/scripts/components/common/form/checkable-filter/index.tsx

```diff
@@ -90,7 +91,7 @@ export default function CheckableFilters(props: CheckableFiltersProps) {
     globallySelected,
     tagItemCleared
   } = props;
-  const [show, setShow] = useState<boolean>(true);
+  const [show, setShow] = useState<boolean>(false);
   const [count, setCount] = useState<number>(0);
```

Comment:
(Not introduced by PR) - `count` and `selected` are not syncing with the state correctly. Ex. On this page: https://deploy-preview-989--veda-ui.netlify.app/data-catalog?search=&taxonomy=%7B%22Topics%22%3A%5B%22air-quality%22%2C%22experimental%22%2C%22agriculture%22%2C%22our-planet%22%2C%22climate%22%5D%7D if I select one more item from Topics filter, I get a message that only 1 topic is selected.

![Screen Shot 2024-06-05 at 11 47 08 AM](https://github.com/NASA-IMPACT/veda-ui/assets/4583806/cfa9721b-4f06-47a0-826a-24b5b6bc82e1)

Reply:
Fixed anyway.